zip-bolt-support.rst

```diff
@@ -0,0 +1,334 @@
+``<MERCH-PK>, <MERCH-SK>``.
+
+The customer key pair is specific to the channel and must not be reused. The merchant key pair is long term and should be used for all customer channels. 
+
+1.2 Wallets
+-------------
+A Bolt channel allows a customer to make or receive a sequence of payments off chain. These payments are tracked and validated using a sequence of *wallets*. A wallet consists of the customer's public key (which ties the wallet to the channel), a wallet-specific public key (which can be from any suitable signature scheme), denoted ``<wpk>``, and the current customer and merchant balances.
+
+After each payment, the customer receives an updated wallet and blind signatures from the merchant on the wallet contents allowing channel close as specified below.
+
+1.2 Opening a Channel: Overview
+-------------
+To open a channel, the customer and merchant exchange key information and set the channel token ``<channel-token> = <cust-pk>, <merch-pk>, <MERCH-PK>``. 
+
+They agree on their respective initial balances ``initial-cust-balance`` and ``initial-merch-balance``.
+
+The customer picks an inital wallet public key ``<wpk>``.
+
+The customer and merchant escrow the necessary funds in a funding transaction, denoted ``escrow-tx``. 
+
+1.3 Closing a Channel: Overview
+-------------
+
+A customer should be able to close the channel by posting a *closing token* ``close-token``, which is a blind signature from the merchant under ``<MERCH-PK>`` on a special closing wallet that contains ``<cust-pk>, <wpk>, <balance-cust>, <balance-merch>, CLOSE``. We use ``cust-close-tx`` to denote the transaction posted by the customer to initiate channel closure.
+
+A merchant should be able to close the channel by either posting a special closing transaction ``merch-close-tx`` (detailed in Section 2.3.2) or, if the customer posts an outdated version of their closing token, a signed revocation token, ``revocation-token`` as detailed below. The revocation token ``revocation-token`` is a signature under the wallet public key ``<wpk>`` on the special revocation message ``<wpk> || REVOKED``. The transaction posted by the merchant to dispute is denoted ``dispute-tx``.
+
+The customer and merchant may also negotiate off-chain to form a *mutual close transaction*, ``mutual-close-tx``. Off-chain collaboration to create ``mutual-close-tx`` reduces the required number of on-chain transactions and eliminates the time delays.
+
+2. Transparent/Shielded Tx: Using T/Z-addresses and WTPs
+-------------
+
+We assume the following specific features are present:
+
+(1) Support for whitelisted transparent programs (WTPs) that enables 2-of-2 multi-sig style transactions
+(2) Can specify absolute lock time in transaction
+(3) Can specify relative lock time in transparent program
+(4) Can specify shielded inputs and outputs
+(5) A non-SegWit approach that fixes transaction malleability
```

Comment:
```suggestion
(5) A non-SegWit approach that fixes transaction malleability
```
Why non-SegWit? I.e. what are the requirements on the non-malleability approach?